packages/dataviz/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Are we in Apache licence on TUI ? 🤔

Reply:
used the same license as faceted search and components